cli/generators/crud/crudGenerator.js

```diff
@@ -57,4 +57,43 @@ module.exports = class extends Generator {
       }
     );
   }
+
+  updateRoutes(templatePath, destinationPath, data) {
+    let fs = this.fs;
+    ejs.renderFile(templatePath, data, (err, renderedTemplate) => {
+      if(err) throw err;
+
+      fs.copy(
+        destinationPath,
+        destinationPath,
+        {
+          process: (content) => {
+            var tree = program(content.toString(), null, {sourceType: 'module', plugins: { jsx: true } });
+            var currentRoutes = tree.var('defaultExport').value().children;
+            var currentImports = tree.body.node;
+
+            var renderedTemplateTree = program(renderedTemplate, null, {sourceType: 'module', plugins: { jsx: true } });
+            var newRoutes = renderedTemplateTree.var('defaultExport').value().children;
+            var newImports = renderedTemplateTree.body.node.filter(function(node) {
+              return node.type === 'ImportDeclaration';
+            });
+
+            newImports.forEach(function(newImport) {
+              currentImports.splice(4, 0, newImport);
+            });
+            
+            var insertRouteAt = currentRoutes.findIndex(function(route){
+              return route.openingElement && route.openingElement.name.name === "IndexRoute";
+            });
+
+            newRoutes.forEach(function(newRoute) {
+                currentRoutes.splice(++insertRouteAt, 0, newRoute);
+            });
+
+            return tree.toString();
```

Comment:
Is modifying the `currentRoutes` variable enough? Don't you need to re-add it to the tree?